CDTDatastore/query/CDTQProjectedDocumentRevision.h

```diff
@@ -28,12 +30,14 @@
 /**
  Initialise with a datastore so mutableCopy can return a full document.
  */
-- (id)initWithDocId:(NSString *)docId
-         revisionId:(NSString *)revId
-               body:(NSDictionary *)body
-            deleted:(BOOL)deleted
-        attachments:(NSDictionary *)attachments
-           sequence:(SequenceNumber)sequence
-          datastore:(CDTDatastore *)datastore;
+- (nullable instancetype)initWithDocId:(NSString *)docId
```

Comment:
Good find. I think that as Apple themselves aren't annotating `init` as `nullable`, I'd definitely prefer we remove these unnecessary `nullable` annotations.